### DISCUSSION TITLE
is it possible to open position at open and close the position at close the same day?

### DISCUSSION THREAD

**Discussion by quantcn** [cvxgrp/cvxportfolio#172] [Q&A]
is it possible to open position at open and close the position at close the same day?

**Comment by enzbus** [comment]
Hello! Yes, although it is not too easy to set up I'm afraid. We would have 2 trading periods per trading day, at market open and at market close. Market returns would have to be intraday period for the first and overnight period for the second. Market volumes it's a bit unclear, ideally you'd want open auction volumes and close auction volumes, but I suspect those are not too easy to get. Anyway if it's just returns you can probably assemble them from the `cvx.YahooFinance` output. Then you'd have to make sure your policy objects are aware of whether you are at the open or at the close time. In short, yes, but it's a very custom usecase and you'd have to be quite experienced to make it work.
